lib/skunk/cli/options/argv.rb

```diff
@@ -19,6 +25,10 @@ def parse # rubocop:disable Metrics/MethodLength
               self.mode = :compare_branches
             end
 
+            opts.on("-o", "--out FILE", "Output report to file") do |filename|
+              @output_filename = String(filename)
```

Comment:
Why we need `String(filename)` ?

Reply:
I was simply following the pattern being used in other cases, for instance when dealing with the `--branch` option a couple of lines above. I admit both seem redundant.

Reply:
I don't think its necessary but maybe @etagwerker had a good reason to do it?

Reply:
@bronzdoc @manuca Good point. I don't think this is necessary.